geode-core/src/distributedTest/java/org/apache/geode/management/DiskManagementDUnitTest.java

```diff
@@ -394,7 +394,7 @@ private MemberMXBean awaitMemberMXBeanProxy(final DistributedMember member) {
     return service.getMBeanProxy(objectName, MemberMXBean.class);
   }
 
-  private void await(final AsyncInvocation createPersistentRegionAsync)
+  private void await(final AsyncInvocation<?> createPersistentRegionAsync)
```

Comment:
This can be `AsyncInvocation<Void>`

Reply:
While it could be, since this method only calls `await(..)`, there is no reason it needs to be `Void` specifically. My preference is to leave method arguments more generic where possible while making return types and variables as specific as possible.

Reply:
Fair enough, I'll just inline it.